src/main/resources/templates/fragments/header.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" class="govuk-template">
+<head>
+    <title>Header</title>
+</head>
+<body class="govuk-template__body">
+<th:block th:fragment="header">
+    <a href="#content" class="govuk-skip-link">Skip to main content</a>
+    <header role="banner" id="global-header" class="govuk-header">
+        <div class="govuk-header__container govuk-width-container">
+            <div class="header-global">
+                <div class="govuk-header__logo">
+                    <a href="/" title="Go to the companies house homepage" id="logo" class="govuk-header__link govuk-header__link--homepage">
```

Comment:
Should `companies house` be capitalised?
```suggestion
                    <a href="/" title="Go to the Companies House homepage" id="logo" class="govuk-header__link govuk-header__link--homepage">
```